### PR TITLE
Delegate related projects to awesome-badges list

### PR DESCRIPTION
**Pull request by paulmelnikow**
As discussed at the core team meeting, we've decided to move the list of related projects into a [separately maintained awesome list](https://github.com/badges/awesome-badges). This PR replaces the "Related projects" section of the readme with a link to that project.

### CODE REVIEW

**Review by calebcartwright — COMMENTED** [review]
lgtm, one wording question inline and also would it be worth finding a way to move https://github.com/badges/shields/blob/master/spec/users.md over to the awesome list in a follow up or does it still make sense to maintain that list here in-repo?

**Comment by paulmelnikow** [comment]
> would it be worth finding a way to move https://github.com/badges/shields/blob/master/spec/users.md over to the awesome list in a follow up or does it still make sense to maintain that list here in-repo?

Heh, I went looking for that list and was thinking we'd decided not to maintain it anymore. I feel like it's a bit more about the spec than anything else…

File: README.md [hunk 1]
```diff
@@ -156,6 +156,21 @@ There is documentation about [hosting your own server][self-hosting].
...
+
+[![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
+
+Status badges are used widely across open-source and private software projects.
+Academics have studied the "signal" badges provide about software project
+quality. There are many existing libraries for rendering these badges, and
+alternatives to the hosted Shields badge service. [awesome-badges][] is a
+curated collection of such resources.
+[Contributions][contributing to awesome-badges] may be considered there.
+(Shields does not endorse these projects.)
```
[the remark this answers is not in the document]
Comment (paulmelnikow) [reply README.md:169]: Yea, good shout.

**Review by calebcartwright — APPROVED** [review]
Sounds good on the spec-related list

**Pull request merged**